adapters/ion-adapter/src/main/java/com/connexta/replication/adapters/ion/IonNodeAdapter.java

```diff
@@ -124,6 +124,8 @@ public boolean createResource(CreateStorageRequest createStorageRequest) {
     HttpHeaders headers = new HttpHeaders();
     headers.set("Accept-Version", "0.1.0");
     headers.setContentType(MediaType.MULTIPART_FORM_DATA);
+    headers.set(
+        "Last-Modified", resource.getMetadata().getMetadataModified().toInstant().toString());
```

Comment:
❓ I know this gives us an ISO-8601 representation, but does that mean we get an ISO offsetdatetime string? We expect DateTimeFormat.ISO.DATE_TIME from org.springframework.format.annotation.

Reply:
Apparently instant is basically an offsetdatetime with no offset. https://stackoverflow.com/questions/32437550/whats-the-difference-between-instant-and-localdatetime. If I use OffsetDateTime.parse().toString() it works and prints out the same.